pallets/xcmp-queue/src/lib.rs

```diff
@@ -447,21 +328,25 @@ pub struct QueueConfigData {
 	/// message sending may recommence after it has been suspended.
 	resume_threshold: u32,
 	/// The amount of remaining weight under which we stop processing messages.
+	#[deprecated(note = "Will be removed")]
 	threshold_weight: Weight,
 	/// The speed to which the available weight approaches the maximum weight. A lower number
 	/// results in a faster progression. A value of 1 makes the entire weight available initially.
+	#[deprecated(note = "Will be removed")]
 	weight_restrict_decay: Weight,
 	/// The maximum amount of weight any individual message may consume. Messages above this weight
 	/// go into the overweight queue and may only be serviced explicitly.
+	#[deprecated(note = "Will be removed")]
 	xcmp_max_individual_weight: Weight,
 }
 
 impl Default for QueueConfigData {
 	fn default() -> Self {
+		#![allow(deprecated)]
 		Self {
-			suspend_threshold: 2,
-			drop_threshold: 5,
-			resume_threshold: 1,
+			suspend_threshold: 200,
```

Comment:
Just wanted to clarify to understand it better (also relevant for bridges).

You've said (in the PR description): "Bumped the default suspension, drop and resume thresholds". But is it really a bump? It is hard to compare, because previously they were measured in pages and now it is individual XCM messages. Page size is `102_400` bytes e.g. at asset hubs (unless I miss something). So previously pallet was fine with `2 * 102_400 = 204_800` bytes of concatenated XCM messages before suspension has been activated. Now it suspends when there are `200` XCM messages enqueued. So if that's really a bump, then the assumption is that XCM messages are larger than `204_800 / 200 = 1024` bytes, right? If they're smaller than `1Kb` (average), then it isn't really a bump :)

Reply:
Thresholds are now checked from the  `pallet-message-queue`' `OnQueueChanged` implementation, where the count is the count of messages, not pages. So yep - it is not measured in messages, not pages. I was mostly interested in seeing actual configuration thresholds here, so all looks fine to me.